setup.py

```diff
@@ -92,7 +92,7 @@
 MAINTAINER_EMAIL = AUTHOR_EMAIL
 LICENSE = 'GPL v3'
 URL = 'https://sk1project.net'
-DOWNLOAD_URL = URL
+DOWNLOAD_URL = 'https://github.com/sk1project/uniconvertor/releases'
```

Comment:
We don't use github as a download service because our CI/CD uploads build artifacts on project site daily.

Reply:
Okay, so I change it to `https://sk1project.net/sk1/download/`?!

Reply:
For UniConvertor this should be ```https://sk1project.net/uc2/download/``` for latest stable release and ```https://sk1project.net/uc2/daily-builds/``` for daily builds.